### ISSUE TITLE
Improve tagging experience

### ISSUE THREAD

**Issue by aluhrs13** [aluhrs13/TheMiniIndex#179]
- [x] Enable tag suggestions for everyone
- [x] Guide users to better tagging on the tagging page
- [x] Link out to https://github.com/aluhrs13/TheMiniIndex/wiki/How-to-tag-Minis
- [x] Split details and tagging pages for Minis

Async refresh of the tag list when adding a tag:
- [x] Make an API endpoint for getting the tags on a Mini.
- [x] Loading spinner on adding/removing a tag
- [x] Call that endpoint when adding or removing a tag is completed.
- [x] Loading spinner when waiting on that call
- [x] Update the tag area to have the latest data.
- [x] Add a spinner icon

Nice to have:
- [x] Call out that we'll be automatically adding tags based on what you add.
- [x] Initial tag load should use the AJAX above.
- [ ] Call out missing "required" tag categories
- [ ] Give the submitter some control over if "their" Minis can be removed.
- [x] Add Office Form to report an issue with a Mini or tagging
- [x] Make the different Statuses clear on the view and edit pages
- [ ] Icons for tag categories?
